CMakeLists.txt

```diff
@@ -633,6 +633,10 @@ if(GCOV)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fprofile-arcs -ftest-coverage")
 endif()
 
+if(KEEP_LOCAL_SYMBOLS)
```

Comment:
Since this only affects the assembler should the flag be `KEEP_LOCAL_ASM_SYMBOLS`?

Reply:
I changed it to `KEEP_ASM_LOCAL_SYMBOLS`.